nexus/db-queries/src/db/datastore/deployment/external_networking.rs

```diff
@@ -783,7 +796,13 @@ mod tests {
             assert_eq!(db_nexus_nics[0].vpc_id, NEXUS_VPC_SUBNET.vpc_id);
             assert_eq!(db_nexus_nics[0].subnet_id, NEXUS_VPC_SUBNET.id());
             assert_eq!(*db_nexus_nics[0].mac, self.nexus_nic.mac);
-            assert_eq!(db_nexus_nics[0].ip, self.nexus_nic.ip.into());
```

Comment:
Cannot comment on it directly, but i think this function should be marked with `#[cfg(test)]`. The initial sight of all these asserts was a bit terrifying.